examples/Buttons/BorderlessButton.md

```diff
@@ -79,6 +79,51 @@ import { faTrash } from '@fortawesome/free-solid-svg-icons';
   }}
   variant={VARIANT.DANGER}
 >
-  <FontAwesomeIcon icon={faTrash} size="small" />
+  <FontAwesomeIcon icon={faTrash} size="lg" />
 </BorderlessButton>
 ```
+
+### With the `forwardRef`
+Ref example: The optional `forwardRef` property is set. When the primary themed button is clicked, the focus shifts to the borderless button.
+```jsx
+import {
+  useState,
+  useRef,
+} from 'react';
+import {
+  Button,
+  VARIANT,
+  } from 'mark-one';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTrash } from '@fortawesome/free-solid-svg-icons';
+
+const BorderlessButtonRefExample = () => {
+  const ref = useRef(null);
+  const [value, setValue] = useState('');
+  const onButtonClick = () => {
+    setTimeout(() => ref.current.focus());
+  }
+  return (
+    <>
+    <Button
+      onClick={onButtonClick}
+      variant={VARIANT.PRIMARY}
+    >
+      Focus the Borderless Button
+    </Button>
+    <BorderlessButton
+      onClick={function() {
+        alert('You clicked the borderless button in which variant equals VARIANT.DANGER')
+      }}
+      variant={VARIANT.DANGER}
+      tabIndex={0}
+      forwardRef={ref}
+      alt="testing"
```

Comment:
I prefer `alt="testing delete"` or `alt="delete"`

Reply:
Ok, that makes sense - updated this to "testing delete" through e8fe7be.